doc/04-Upgrading.md

```diff
@@ -3,6 +3,65 @@
 Specific version upgrades are described below. Please note that version upgrades are incremental.
 If you are upgrading across multiple versions, make sure to follow the steps for each of them.
 
+## Upgrading to Icinga DB v1.1.2
+
+Please apply the `1.1.2.sql` upgrade script to your database. For package installations, you can find this file at
+`/usr/share/icingadb/schema/mysql/upgrades/` or `/usr/share/icingadb/schema/pgsql/upgrades/`, depending on your
+database vendor.
+
+!!! warning
+
+    Please ignore this warning if you are using PostgreSQL. This applies to MySQL/MariaDB users only.
+
+    In order to fix a loading performance issue of the history view in Icinga DB Web, this upgrade script adds an
+    appropriate index on the `history` table. Creating this new index normally takes place without blocking any other
+    queries. However, this may hang for a relatively considerable time, blocking all Icinga DB queries on all
+    `*_history` tables and the `history` table inclusively if there is an ongoing, long-running query on the `history`
+    table. One way of causing this to happen is if an Icinga Web user accesses the `icingadb/history` view just before
+    you are running this script. Depending on how many entries you have in the history table, Icinga DB Web may take
+    quite a long time to load, until your web servers timeout (if any) kicks in. 
+    See [unblock history tables](#unblock-history-tables) for details.
+
+### Unblock History Tables
```

Comment:
So far, within this section, the only hint that this only applies to MySQL/MariaDB is the "MariaDB [icingadb]>" prompt. Maybe just add "(MySQL/MariaDB only)" to its title or add an [`!!! info` block](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types).